source/tutorials/smb-server.rst

```diff
@@ -0,0 +1,134 @@
+.. _clear-samba-share-to-Windows:
+
+Setup Samba Server
```

Comment:
```suggestion
Samba* Server
```

The goal of the titling here was to have it be similar to the the client setup document: `"Samba* as a client`. All the tutorial titles are also heavily simplified to have the noun without verbs. This would be a deviation. 

See https://docs.01.org/clearlinux/latest/tutorials/index.html#tutorials

Reply:
done.